ig/README.md

```diff
@@ -32,3 +33,24 @@ A Dockerfile is provided that can be used to generate the Implementation Guide.
 1. The container also pre-generates the documentation.  This process could make the image build take 10-15 minutes
 1. All of the generated documentation should be in the `/trusted-intermediary/output` directory
 1. If building locally, you may want to add a `--no-cache` to the `docker buildx build` command so that the documentation generation is not cached
+
+## Using Docker for local development
+
+Provided is a Dockerfile suitable for running the implementation guide documentation generation without the need to install all the tooling locally.
+The Docker image needs to be built and then run with an appropriate `mount` flag in order to mount your local `ig` directory into the running Docker container.
+This will allow bi-directional writes so that the Docker container can generate the documentation, and as a developer you can view the generated documentation easily.
+A developer will also be able to update any input files for the documentation and re-run the documentation generation without any intermediate steps.
+
+1. In a terminal, navigate to this `ig` directory
+2. run `docker buildx build -t ig-local-dev -f Dockerfile-local-dev .` to build the Docker container
```

Comment:
Is `buildx` really needed?  You can just do `build`?

Reply:
From my read of the Docker documentation, `buildx` is needed for the copy with permissions that's used in `Dockerfile`.  In theory `build` should work fine for `Dockerfile-local-dev`.  I can test it out.